pages/linux/dconf.md

```diff
@@ -0,0 +1,16 @@
+# dconf
+
+> Simple tool for manipulating a dconf database.
```

Comment:
```suggestion
> Simple tool for manipulating dconf databases.
> See also `dconf write`.
```
I think the plural generally sounds better in descriptions like this. Also, as long as we only have one sub-command page, I think we can cross-reference them.

Reply:
>Also, as long as we only have one sub-command page, I think we can cross-reference them.

Sounds like a good idea!

Reply:
> as long as we only have one sub-command page, I think we can cross-reference them.

I was wondering about this when I made the PR, I looked for any other page that referenced the subcommand and couldn't find any, so I figured it wasn't advised. It might be a good idea to add that to the documentation for pages with a single sub-command :rocket: